src/components/utils/responsive-props.spec.js

```diff
@@ -0,0 +1,79 @@
+import {generateResponsiveClassNames} from './responsive-props';
```

Comment:
@clxandstuff 
I am not sure that this test is enough. We should always test public api in the first place, and while this helper might be working correctly, it doesn't mean it's being used correctly in components. 

I am aware that flex component has much better coverage in this regard. Maybe outside of this PR we may consider our approach to testing responsive props on components level so we can incorporate better solution in the future.

Reply:
I agree. There are a lot of tests to cover, but I think I could write helper to make it less bloated. I'll give it a try in this PR.

Reply:
@coderitual check `Flex.spec.js file` how tests look. I'm gonna wrap common logic to helper but first would like to confirm if this is good enough?